### PR TITLE
new faq for api token rotation and find user id

### PR DESCRIPTION
**Pull request by mirnawong1**
created a new faq to address [slack thread](https://dbt-labs.slack.com/archives/C02NCQ9483C/p1672957302892219) and also add'l faq to inform user how to find user id

also added faq in the [user token docs page](https://docs.getdbt.com/docs/dbt-cloud-apis/user-tokens) to improve and expedite user experience

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/faqs/API/rotate-token.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: How can I rotate my user API token?
+description: "Instructions on how to swap API token"
```
Comment (matthewshaver) [inline website/docs/faqs/API/rotate-token.md]: Does this need to include the word "rotate"? "Swap" isn't used anywhere else in this article but if it helps with SEO I'm on board.

File: website/docs/faqs/Accounts/find-user-id.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: Where can I find my user id?
+description: "Instructions on where to find your user id"
+sidebar_label: 'Where can I find my user id'
+id: find-user-id
+---
+
+You may need to find your user id to rotate your API token, and so on. Refer to the following steps to find your user id in dbt Cloud:
```
Comment (matthewshaver) [inline website/docs/faqs/Accounts/find-user-id.md]:
```suggestion
Refer to the following steps to find your user ID in dbt Cloud:
```

File: website/docs/faqs/API/rotate-token.md (hunk 1 above)
Comment (mirnawong1) [reply website/docs/faqs/API/rotate-token.md]: d'oh! good callout!

Comment (mirnawong1) [inline website/docs/faqs/API/rotate-token.md]:
```suggestion
description: "Instructions on how to rotate API token"
```

File: website/docs/faqs/Accounts/find-user-id.md (hunk 2 above)
Comment (matthewshaver) [reply website/docs/faqs/Accounts/find-user-id.md]: Unless we want to expound on "and so on" the first part of this feels a little excessive.

File: website/docs/faqs/API/rotate-token.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+
+3. Find the new key in the API response or in dbt Cloud by going to **Account Settings** and then **API Access**.
+
+
+**dbt Cloud deployment**
+
+If your [dbt Cloud deployment](/docs/deploy/regions-ip-addresses) uses a different access URL, replace `cloud.getdbt.com` with the URL of your instance. For example:
+
+- For Virtual Private dbt &mdash; Use `http://cloud.customizedurl.getdbt.com/`, not `http://cloud.getdbt.com/`<br />
```
Comment (matthewshaver) [inline website/docs/faqs/API/rotate-token.md]: Adding for clarity as dbt itself is not a product. 
```suggestion
- For Virtual Private dbt Cloud &mdash; Use `http://cloud.customizedurl.getdbt.com/`, not `http://cloud.getdbt.com/`<br />
```

File: website/docs/faqs/Accounts/find-user-id.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
+title: Where can I find my user id?
```
Comment (matthewshaver) [inline website/docs/faqs/Accounts/find-user-id.md:2]: Maybe we should discuss this in the next meeting - from my experience ID should always be capitalized unless it's referring to a field where it's lowercase - like `user_id`

File: website/docs/faqs/API/rotate-token.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+3. Find the new key in the API response or in dbt Cloud by going to **Account Settings** and then **API Access**.
+
+
+**dbt Cloud deployment**
+
+If your [dbt Cloud deployment](/docs/deploy/regions-ip-addresses) uses a different access URL, replace `cloud.getdbt.com` with the URL of your instance. For example:
+
+- For Virtual Private dbt Cloud &mdash; Use `http://cloud.customizedurl.getdbt.com/`, not `http://cloud.getdbt.com/`<br />
+- For EMEA region &mdash; Use `https://emea.dbt.com/`, not `http://cloud.getdbt.com/`, and so on.
```
Comment (matthewshaver) [inline website/docs/faqs/API/rotate-token.md]: Would Virtual Private Cloud users in EMEA follow the `http://cloud.customizedurl.getdbt.com/` example or would it be `http://emea.customizedurl.getdbt.com/`

**Review by matthewshaver — DISMISSED** [review]

File: website/docs/faqs/API/rotate-token.md (hunk 5 above)
Comment (mirnawong1) [reply website/docs/faqs/API/rotate-token.md]: almost certain they would customize their url and won't use the emea bit but am confirming!

File: website/docs/faqs/Accounts/find-user-id.md (hunk 4 above)
Comment (mirnawong1) [reply website/docs/faqs/Accounts/find-user-id.md:2]: Def something to confirm, I didn't know that and I think I was using lower case and upper case interchangeably but would be good to confirm. I've updated that one - thanks @matthewshaver !

File: website/docs/faqs/API/rotate-token.md (hunk 5 above)
Comment (mirnawong1) [reply website/docs/faqs/API/rotate-token.md]: ok i've confirmed with Will and Brian that vpc users would have the first URL as it's ST. if they're MT in emea, then they would have https://emea..

**Comment by mirnawong1** [comment]
hey @matthewshaver! i've made some more tweaks to this now - can you take another gander please? thank you

**Review by matthewshaver — DISMISSED** [review]

**Review by matthewshaver — APPROVED** [review]

**Pull request merged**